scripts/benchmarking/run_all_benchmarks.sh

```diff
@@ -1,165 +0,0 @@
-#!/usr/bin/env bash
```

Comment:
why the file was deleted?

Reply:
oops, was done accidentally
restored in [8074e3d](https://github.com/gear-tech/gear/pull/1784/commits/8074e3df085c5c796da3ee150a1474a2c4e36897)